<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>a3k</class>
 <widget class="QMainWindow" name="a3k">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>a3k</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="stepButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QListWidget" name="opList">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>61</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="insnList">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>81</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="addrList">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>81</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>174</width>
      <height>293</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>R0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reg_0">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>R1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reg_1">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>R2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reg_2">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>R3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reg_3">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>R4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reg_4">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>R5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reg_5">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>R6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reg_6">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>R7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reg_7">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>R8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reg_8">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="fastButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Fast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop/Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pause">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No File Loaded</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuRun_Speed_2">
    <property name="title">
     <string>Run Speed</string>
    </property>
    <addaction name="action50"/>
    <addaction name="action100_2"/>
    <addaction name="action200_2"/>
    <addaction name="action300_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun_Speed_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_File">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionChange_Speed">
   <property name="text">
    <string>Change Speed</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action100_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </action>
  <action name="action200_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </action>
  <action name="action300_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>300</string>
   </property>
  </action>
  <action name="action50">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
